controllers/storageclass/suite_test.go

```diff
@@ -104,6 +104,8 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	err = os.Setenv(util.ExporterImageEnvVar, "docker.io/ibmcom/ibm-storage-odf-block-driver:v0.0.22")
 	Expect(err).NotTo(HaveOccurred())
+	err = os.Setenv(util.OperatorPodNameEnvVar, "ibm-storage-odf-operator*")
```

Comment:
why `*` at the end? is it a typo? same for pv suite

Reply:
typo from old tries .. .
see now